eclair-core/src/test/scala/fr/acinq/eclair/wire/ChannelCodecsSpec.scala

```diff
@@ -137,6 +137,18 @@ class ChannelCodecsSpec extends FunSuite {
   }
 
   test("encode/decode htlc") {
+    // these encoded HTLC were produced by a previous version of the codec (at commit 8932785e001ddfe32839b3f83468ea19cf00b289)
```

Comment:
Can you extract that (backwards-compatibility check) to a separate test?

Reply:
done

Reply:
+1, and also clearly label it as a non-reg test